tensorboard/webapp/core/state.ts

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {State as AppRoutingState} from '../app_routing/store/app_routing_types';
+import {State as FeatureFlagState} from '../feature_flag/store/feature_flag_types';
+import {State as CoreState} from './store/core_types';
+
+export type State = CoreState & AppRoutingState & FeatureFlagState;
```

Comment:
Seems like we need further discussion with the team before merging this part of the PR.

Reply:
For now, I took the easy route of re-exporting the symbol. I can undo all these changes but... a bit lazy :)